Capri.Web/src/router.ts

```diff
@@ -0,0 +1,48 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+import LoginView from './components/loginView.vue';
+import CardsView from './components/cardsView.vue';
+import MyProposals from './components/myProposals.vue';
+import PromoterList from './components/promotersList.vue';
+import Import from './components/importPromoters.vue';
+import axios from 'axios';
+import VueAxios from 'vue-axios';
+
+Vue.use(Router);
+Vue.use(VueAxios, axios);
+
+Vue.axios.interceptors.request.use(
```

Comment:
Move that after assinging `router` variable.

Reply:
Moved.